code/index.html

```diff
@@ -2,21 +2,301 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>The Technigo Times</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:ital@0;1&family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">    <link rel="icon" type="image/svg+xml" href="resources/favicon.svg">
   </head>
   <body>
     <header>
-      <!-- Some kind of logo and navbar -->
+      <div class="left">
+        <div class="hamburger"></div>
```

Comment:
Oh dang it! :) Not sure why i missed that because it worked earlier, but it is in css line 404, it works when changed from  `.left > .hamburger,` to `header > .left > .hamburger,`
Thats life!